launcher.py

```diff
@@ -258,6 +260,12 @@ def get_parser_args():
         type=float,
         help='Lower bound of heterozygous SNPs (%). Samples with lower values are removed from the relatedness detection analysis.')
 
+    parser.add_argument(
+        '--seed',
+        default=randint(0, 10**7),
+        type=int,
+        help='Pedsim pedigree structures simulation integer seed. Seed is random by default.')
```

Comment:
```suggestion
        help='Random seed to initialise Ped-sim pedigree structure for the simulation. The default value is randomly generated.')
```

Reply:
I think the word "structure" in "initialize Ped-sim pedigree structure" make it seem like structure of pedigree simulation is determined by random seed. However it is defined by .def file. I suggest we just remove the word from arg description.